### PR TITLE
mention migration + update copyright

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,9 +48,11 @@ For development:
 Thanks
 ------
 
-PuppetModule.info is maintained by Dominic Cleal, and is unaffiliated to Puppet.
+PuppetModule.info was maintained by Dominic Cleal, and is unaffiliated to Puppet.
```
Comment (ekohl) [inline README.md:51]:
```suggestion
PuppetModule.info was authored and maintained by Dominic Cleal, and is unaffiliated to Puppet.
```

File: README.md [hunk 2]
```diff
@@ -48,9 +48,11 @@ For development:
...
 RubyDoc.info was created by Loren Segal (YARD) and Nick Plante (rdoc.info) and is a project of DOCMETA, LLC.
 Additional help was provided by [YARD's friendly developer community](https://github.com/lsegal/rubydoc.info/graphs/contributors).
+
+Dominic run the site for years, until he migrated it to Vox Pupuli.
```
Comment (arjenz) [inline README.md:55]: run -> ran

File: README.md [hunk 3]
```diff
@@ -48,9 +48,11 @@ For development:
...
 Pull requests welcome!
 
-(c) 2015 DOCMETA LLC, 2017 Dominic Cleal. This code is distributed under the MIT license.
+(c) 2015 DOCMETA LLC, 2017 Dominic Cleal, 2022 Vox Pupuli. This code is distributed under the MIT license.
```
Comment (arjenz) [inline README.md:58]: It's 2023 now, or is 2022 intentional?

File: README.md [hunk 4]
```diff
@@ -23,11 +23,11 @@ enhancements are welcome.
...
 
 ```
-$ git clone git://github.com/domcleal/puppetmodule.info
-$ cd puppetmodule.info
-$ bundle install
-$ rake gems:update
-$ rackup config.ru
+git clone git://github.com/voxpupuli/puppetmodule.info
+cd puppetmodule.info
+bundle install --path .vendor --jobs $(nproc)
```
Comment (ekohl) [inline README.md:28]: `--path` is obsolete and we should use `bundle config path .vendor` (from the top of my head).

File: README.md [hunk 5]
```diff
@@ -23,11 +23,11 @@ enhancements are welcome.
...
+rake gems:update
```
Comment (ekohl) [inline README.md:29]: I really wonder what this does.I really wonder what this does. I wonder if this should really be `modules:update` instead.

File: README.md [hunk 6]
```diff
@@ -48,9 +48,11 @@ For development:
 Thanks
 ------
 
-PuppetModule.info is maintained by Dominic Cleal, and is unaffiliated to Puppet.
+PuppetModule.info was maintained by Dominic Cleal, and is unaffiliated to Puppet.
 RubyDoc.info was created by Loren Segal (YARD) and Nick Plante (rdoc.info) and is a project of DOCMETA, LLC.
 Additional help was provided by [YARD's friendly developer community](https://github.com/lsegal/rubydoc.info/graphs/contributors).
+
+Dominic ran the site for years, until he migrated it to Vox Pupuli.
```
Comment (ekohl) [inline README.md:55]: migrated or donated?

File: README.md (hunk 4 above)
Comment (bastelfreak) [reply README.md]: updated it. it needs to have `--local` if we want to only set the option for the local project.

File: README.md (hunk 5 above)
Comment (bastelfreak) [reply README.md]: indeed it's `modules:update`. Fixed it.

File: README.md (hunk 6 above)
Comment (bastelfreak) [reply README.md]: I'm not sure what's the actual difference. I updated it to donated. let me know what you think about it.

File: README.md [hunk 7]
```diff
@@ -23,11 +23,13 @@ enhancements are welcome.
...
 ```
-$ git clone git://github.com/domcleal/puppetmodule.info
-$ cd puppetmodule.info
-$ bundle install
-$ rake gems:update
-$ rackup config.ru
+git clone git://github.com/voxpupuli/puppetmodule.info
+cd puppetmodule.info
+bundle config --local path .vendor/
+bundle config --local jobs $(nrpoc)
```
Comment (ekohl) [inline README.md:29]:
```suggestion
bundle config --local jobs $(nproc)
```

Though perhaps it's redundant now? https://bundler.io/v2.4/man/bundle-config.1.html writes:
> jobs (BUNDLE_JOBS): The number of gems Bundler can install in parallel. Defaults to the number of available processors.

Looking through the changelog this is a Bundler 2.2 feature.

File: templates/layout.erb [hunk 8]
```diff
@@ -136,16 +136,16 @@
...
         </small>
       </p>
     </div>
 
     <%= yield %>
 
     <div id="footer">
-      <p><a href="https://github.com/domcleal/puppetmodule.info"><%= h settings.name %></a> is powered by <a href="http://yardoc.org/">yard</a> <%= YARD::VERSION %> (ruby-<%= RUBY_VERSION %>)</p>
-      <p>&copy; 2015 DOCMETA, LLC + 2017 Dominic Cleal</p>
+      <p><a href="https://github.com/voxpupuli/puppetmodule.info"><%= h settings.name %></a> is powered by <a href="http://yardoc.org/">yard</a> <%= YARD::VERSION %> (ruby-<%= RUBY_VERSION %>)</p>
```
Comment (ekohl) [inline templates/layout.erb:147]:
```suggestion
      <p><a href="https://github.com/voxpupuli/puppetmodule.info"><%= h settings.name %></a> is powered by <a href="https://yardoc.org/">yard</a> <%= YARD::VERSION %> (ruby-<%= RUBY_VERSION %>)</p>
```

File: README.md (hunk 7 above)
Comment (bastelfreak) [reply README.md:29]: ah nice, so it's finally obsolete.

**Review by arjenz — APPROVED** [review]

**Review by SimonHoenscheid — APPROVED** [review]

**Pull request merged**